torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9505,6 +9506,8 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.skip("Skipped!"), 'TestGradients', 'test_fn_gradgrad'),
                # GradcheckError: gradcheck expects all tensor inputs are dense when check_sparse_nnz is set to False
                DecorateInfo(unittest.skip("Skipped!"), 'TestGradients', 'test_forward_mode_AD'),
+               # https://github.com/pytorch/pytorch/issues/85169
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_compare_cpu'),
```

Comment:
You should be able to remove this xfail now that the stack has been re-ordered. `sparse.sampled_addmm` should not still be failing, if it is there is another bug.

Reply:
Yes, it should be removed the unexpected success is causing the failure of ([linux-bionic-cuda11.6-py3.10-gcc7 / test (default, 1, 4, linux.4xlarge.nvidia.gpu)](https://github.com/pytorch/pytorch/actions/runs/3108192266/jobs/5037447741))

Reply:
removed